Controllers/FileController.cs

```diff
@@ -1,49 +1,220 @@
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Collections.Generic; 
+using System.Globalization;
+
 namespace TodoApi.Controllers
 {
+    public class Author
```

Comment:
These look like datamodel classes, they should be in their own files in the Models directory